core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1031,20 +1030,46 @@ class HttpGoogleServicesDAO(
     }
   }
 
+  override def addGoogleProjectLabel(googleProject: GoogleProjectId, key: String, value: String): Future[Unit] = {
+    implicit val service: GoogleInstrumentedService.Value = GoogleInstrumentedService.CloudResourceManager
+
+    val cloudResourceManagerServiceAccountCredential = getCloudResourceManagerServiceAccountCredential
+    val cloudResourceManager = getCloudResourceManager(cloudResourceManagerServiceAccountCredential)
+    for {
+      project <- getGoogleProject(googleProject)
+      _ = project.getLabels.put(key, value)
+      _ <- executeGoogleRequestWithRetry(cloudResourceManager.projects().update(googleProject.value, project))
+    } yield {
+
+    }
+  }
+
+  override def updateGoogleProjectName(googleProject: GoogleProjectId, name: String): Future[Unit] = {
+    implicit val service = GoogleInstrumentedService.CloudResourceManager
+
+    val cloudResourceManagerServiceAccountCredential = getCloudResourceManagerServiceAccountCredential
+    val cloudResourceManager = getCloudResourceManager(cloudResourceManagerServiceAccountCredential)
+    for {
+      _ <- executeGoogleRequestWithRetry(cloudResourceManager.projects().update(googleProject.value, new Project().setName(name)))
```

Comment:
It looks like we need to do some string validation because the displayname looks pretty limited :/

```
  /**
   * The user-assigned display name of the Project. It must be 4 to 30 characters. Allowed
   * characters are: lowercase and uppercase letters, numbers, hyphen, single-quote, double-quote,
   * space, and exclamation point.
   *
   * Example: My Project Read-write.
   * @param name name or {@code null} for none
   */
  public Project setName(java.lang.String name) {
```